### ISSUE TITLE
Bote v0.4: User guide and FAQ are in english only ? (Trac #1641)

### ISSUE THREAD

**Issue by str4d** [i2p/i2p.i2p-bote#57]
Bote v0.4 User guide and FAQ are in english only. Is it normal or not ? Are the translations planed for later ?

Migrated from https://trac.i2p2.de/ticket/1641
```json
{
    "status": "closed", 
    "changetime": "2015-09-12T15:40:53", 
    "description": "Bote v0.4 User guide and FAQ are in english only. Is it normal or not ? Are the translations planed for later ?\n", 
    "reporter": "hummingbird", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1442072453193080", 
    "component": "apps/plugins", 
    "summary": "Bote v0.4: User guide and FAQ are in english only ?", 
    "priority": "minor", 
    "keywords": "I2P-Bote", 
    "version": "0.9.21", 
    "parents": "", 
    "time": "2015-08-21T06:29:15", 
    "milestone": "0.9.22", 
    "owner": "str4d", 
    "type": "defect"
}
```

**Comment by str4d** [comment]
Trac update at 20150821T11:31:40: 
* str4d commented:

> It's not normal. The Spanish and French translations are included in the WAR, but the JSP to load the correct language is failing (it worked when I implemented it).


* str4d changed owner from "" to "str4d"
* str4d changed status from "new" to "accepted"

**Comment by str4d** [comment]
Trac update at 20150912T15:40:53: 
* str4d commented:

> (Trivial) bug found, fixed in `6f2c7a668a121e5ad78ca970cff5647f2281a5dd`.


* str4d changed milestone from "undecided" to "0.9.22"
* str4d changed resolution from "" to "fixed"
* str4d changed status from "accepted" to "closed"
